tests/component/test_write_properties.py

```diff
@@ -0,0 +1,126 @@
+"""Tests for validating write properties."""
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx.constants import WriteRelativeTo
+from nidaqmx.errors import DaqError, DAQmxErrors
+from nidaqmx.task import Task
+
+
+@pytest.fixture(scope="function")
```

Comment:
scope='function' is the default, so you can omit it.

Reply:
removed.